README.md

```diff
@@ -71,7 +75,8 @@ tests:
 ## License
 Apache 2.0
 
-This is not an officially supported Google product.
+## Support
+This is not an officially supported Google product. If you find a bug, have questions or want to give feedback, add an issue to the [project's issue queue](https://github.com/apigee/devportal-kickstart-drupal-ansible-installer/issues). If you have issues running Ansible or with one of the open source [Ansible roles included](https://github.com/apigee/devportal-kickstart-drupal-ansible-installer/blob/master/requirements.yml) in this project, you can work with those projects to find solutions to your issue.
```

Comment:
This is not an officially supported Google product. The architectural choices of these scripts do not connote support implications. Please refer to the extensive [documentation on drupal.org](https://www.drupal.org/docs/user_guide/en/installation-chapter.html) for guidance on installation.